config/routes.rb

```diff
@@ -74,9 +74,9 @@
 
   # Podcast
   get 'podcast/feed',                           to: 'podcast#feed',        as: :podcast_feed, defaults: { format: 'rss' }
-  get 'podcasts',                               to: 'podcast#index',       as: :podcast
-  get 'podcasts/:slug',                         to: 'podcast#show',        as: :podcast_detail
-  get 'podcasts/:slug/episodes/:id',            to: 'episodes#show',       as: :podcast_episode
+  get 'podcasts',                               to: 'podcast#index',       as: :podcasts
```

Comment:
Maybe in a future PR we should think about use `resources` and `resource` and make the routes as restfull as possible?

Reply:
Totally!